.gitignore

```
@@ -1,13 +1,38 @@
-target/
-**/target/
+!polkadot.service
+.cargo-remote.toml
+.direnv/
+.DS_Store
+.env
```

```suggestion
.env*
```
Some `.env*` files can be sensitive so I would recommend being "aggressive" here by default.